test/commands/log_in_out.test.ts

```diff
@@ -43,4 +43,8 @@ describe('apify login and logout', () => {
 
         expect(isGlobalConfig).to.be.eql(false);
     });
+
+    it('should open browser with interactive login', async () => {
```

Comment:
I'm not sure we should run the tests that open a browser in CI... 👀 (and we can use `it.runIf(<some bool condition here>)()`)